### ISSUE TITLE
copy to clipboard feature is not accessible for screenreader users

### ISSUE THREAD

**Issue by vojtapolasek** [firasdib/Regex101#1470]
## Bug Description

I am running Orca screenreader under Arch Linux, Firefox 84.0.2. Unfortunately, many of functions are missing correct accessibility roles etc... most of them I can workaroud somehow, but the "copy to clipboard" feature, which copies the regex to clipboard, is not accessible at all. That means that I can't navigate to the element and activate it through keyboard.

Please look here for more info about how to improve accessibility of the application.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA

Beside this problem, I like the application much. Some parts are already accessible.

### Reproduction steps

Enter some regex. Well, honestly, usually you can use keyboard (tab character) to move among elements. Here it is very hard. Try to use it, but it does not work.

### Expected Outcome

It should be possible to navigate in this way toward any element on the site.

### Browser
Firefox 84.0.2

### OS
Arch Linux

**Comment by Ouims** [comment]
Hello, this could be related to the specifc tab behavior that was decided in #676 when that new copy to clipboard button was not there, sounds like it could be made to also set focus on that copy to clipboard button.

**Comment by firasdib** [comment]
Adjusted in new version, eta couple of days
